extensions/hardening/rke1/harden_nodes.go

```diff
@@ -0,0 +1,90 @@
+package rke1
+
+import (
+	"strings"
+
+	"github.com/rancher/shepherd/pkg/nodes"
+	"github.com/sirupsen/logrus"
+)
+
+// HardenRKE1Nodes hardens the nodes by setting kernel parameters and creating the etcd user and group
+func HardenRKE1Nodes(nodes []*nodes.Node, nodeRoles []string) error {
+	for _, node := range nodes {
+		logrus.Infof("Setting kernel parameters on node: %s", node.NodeID)
+		_, err := node.ExecuteCommand("sudo bash -c 'echo vm.panic_on_oom=0 >> /etc/sysctl.d/90-kubelet.conf'")
+		if err != nil {
+			return err
+		}
+
+		_, err = node.ExecuteCommand("sudo bash -c 'echo vm.overcommit_memory=1 >> /etc/sysctl.d/90-kubelet.conf'")
```

Comment:
I am not sure of the documented OS, I would say no? I see where you're going with that, I personally don't think that's enough of a concern to not have it here. Our automation typically runs through Ubuntu, RHEL or SLES and each of those don't fit that issue IMO.

Reply:
RKE1/RKE2 share the file names, that's true, but they're in different packages. I ended up putting separate consts in their respective files. I thought it would look cleaner than having to define elsewhere.